development/data-flows-for-analytics.md

```diff
@@ -0,0 +1,85 @@
+# Data flows
+
+In this page we focus on how data flows through the various CHT components. CHT is built to support the delivery of quality community health in hardest to reach areas. It is therefore designed to work in areas with low connectivity making it an offline-first toolkit. The architectural and technology choices in the stack are mostly guided by this principle which will be evident in the discussion of the data management pipeline.
+
+
+## Overview
+
+![overview](img/data-flows-overview-1.png)
+
+![overview](img/data-flows-overview-2.png)
+
+At a high level:
+
+- data is collected from the device of a health worker;
+- this data is pushed to an online instance from where this data is available to other health workers, supervisors and decision makers;
+- the data is transferred to a relational database and presented in forms suitable for analytics and visualizations;
```

Comment:
```suggestion
- the data is transferred to a relational database and made available analytics and visualizations;
```

The word "forms" is overloaded as it's used to mean other things.

Reply:
I'd probably give couch2pg a plug here, to make it clear that it requires a separate component.